packages/manager/modules/vps/src/dashboard/tile/configuration/translations/Messages_fr_FR.json

```diff
@@ -15,5 +15,8 @@
   "vps_dashboard_tile_configuration_upgrade_confirmation_title": "Votre modification a bien été prise en compte.",
   "vps_dashboard_tile_configuration_upgrade_confirmation_details": "Vous allez recevoir un e-mail de confirmation dans un délai maximal de deux heures une fois les changements effectifs sur votre VPS. Un redémarrage de votre machine sera nécessaire pour profiter de votre nouvelle configuration.",
   "vps_dashboard_tile_configuration_upgrade_confirmation_track_order": "Suivre ma commande",
-  "vps_dashboard_tile_configuration_upgrade_complete_reboot_required": "L‘ajout de ressources sur votre VPS a bien été effectué. Veuillez redémarrer votre machine afin d‘activer sa nouvelle configuration."
+  "vps_dashboard_tile_configuration_upgrade_complete_reboot_required": "L‘ajout de ressources sur votre VPS a bien été effectué. Veuillez redémarrer votre machine afin d‘activer sa nouvelle configuration.",
```

Comment:
```suggestion
  "vps_dashboard_tile_configuration_upgrade_complete_reboot_required": "L‘ajout de ressources sur votre VPS a bien été effectué. Veuillez redémarrer votre machine afin d'activer sa nouvelle configuration.",
```

Reply:
Don't include in my modification, I just add ',' in end of line.